server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/sql/Db2SqlParser.g4

```diff
@@ -1737,6 +1737,7 @@ dbs_index_identifier: IDENTIFIER;
 dbs_index_name: T=dbs_sql_identifier {validateLength($T.text, "Index name", 128);};
 dbs_integer: INTEGERLITERAL | SINGLEDIGIT_1 | DOUBLEDIGIT_1 | SINGLEDIGITLITERAL;
 dbs_integer_constant: dbs_integer | NUMERICLITERAL; //range 1 - 32767
+dbs_decimal_constant: dbs_integer DOT_FS dbs_integer?;
```

Comment:
I think this needs to be done in the lexer.
- `SELECT 0 . 1 FROM SYSIBM.SYSDUMMY1` is not valid
- `.1` is valid
- `1e10` is valid